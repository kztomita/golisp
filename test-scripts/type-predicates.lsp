(print (symbolp 'foo))
(print (symbolp 1))
(print (symbolp nil))
(print (symbolp '()))
(print (symbolp "foo"))
(print "====")
(print (stringp "foo"))
(print (stringp 'foo))
(print "====")
(print (consp (cons 1 2)))
(print (consp '(1 2)))
(print (consp nil))
