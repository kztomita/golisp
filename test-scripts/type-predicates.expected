
t 
nil 
t 
t 
nil 
"====" 
t 
nil 
"====" 
t 
t 
nil 
